cmd/tsbs_generate_queries/databases/influx/iot_test.go

```diff
@@ -456,3 +512,43 @@ func TestTenMinutePeriods(t *testing.T) {
 	}
 
 }
+
+func runIoTTestCases(t *testing.T, testFunc func(*IoT, IoTTestCase) query.Query, s time.Time, e time.Time, cases []IoTTestCase) {
+	rand.Seed(123) // Setting seed for testing purposes.
+
+	for _, c := range cases {
+		t.Run(c.desc, func(t *testing.T) {
+			b := BaseGenerator{}
+			dq, err := b.NewIoT(s, e, 10)
```

Comment:
I saw a test above that was supposed to fail because the required number of trucks was 20, and this was greater than the scale. Can we extract this 10 to a const and use it in the test above also?

Reply:
agree ^